cerberus/kubernetes/client.py

```diff
@@ -1,15 +1,17 @@
+import yaml
 import logging
 from collections import defaultdict
 from kubernetes import client, config
 from kubernetes.client.rest import ApiException
 import cerberus.invoke.command as runcommand
-import yaml
+
 
 # Load kubeconfig and initialize kubernetes python client
 def initialize_clients(kubeconfig_path):
-    global cli
+    global cli, pods_tracker
```

Comment:
Maybe initialize pods_tracker outside this function as it's meant for initializing client?